components/Dialogs/docs/theming.md

```diff
@@ -18,11 +26,18 @@ the design parameters that you can use to theme your dialogs.
 // Step 1: Import the DialogThemer extension
 import MaterialComponents.MaterialDialogs_DialogThemer
 
-// Step 2: Create or get a alert scheme
-let alertScheme = MDCAlertScheme()
+// Step 2: Create or get a Material container scheme
```

Comment:
This is a code snippet for Themers (note the import statement on line 27). Please add the Theming Extension docs to a separate section of the documentation.  That way we can keep the Themer documentation until it's removed.